PTGLdynamics/codes/plotting-scripts/heatmapVisualisation.py

```diff
@@ -0,0 +1,252 @@
+########### settings ###########
+
+# This script's version as MAJOR.MINOR.PATCH
+#   major: big (re-)implementation, new user interface, new overall architecture
+#   minor: new functions, git merge
+#   patch: fixes, small changes
+#   no version change: fix typos, changes to comments, debug prints, small changes to non-result output, changes within git branch
+# -> only increment with commit / push / merge not while programming
+version = "2.0.0"  
```

Comment:
Not sure atm: This script was newly written, should it not be 1.0.0 then? But I think parts were copied from elsewhere. Is this the reason for 2.0.0?